fili-core/src/main/java/com/yahoo/bard/webservice/web/apirequest/DataApiRequest.java

```diff
@@ -169,7 +171,9 @@
      * @param dimensionDictionary  DimensionDictionary
      *
      * @return Set of filter objects.
+     * @deprecated Use getFilterGenerator.generate instead
```

Comment:
There doesn't appear to be a `getFilterGenerator` method on this interface. Is that intentional?

Reply:
No, but I'm taking care of it in the next commit.